presc/copies/copying.py

```diff
@@ -0,0 +1,55 @@
+from presc.copies.sampling import labeling
+from presc.copies.evaluations import empirical_fidelity_error
+
+
+class ClassifierCopy:
+    def __init__(
+        self,
+        original,
+        copy,
+        sampling_function,
+        *sampling_parameters,
+        **k_sampling_parameters
+    ):
+        self.original = original
+        self.copy = copy
+        self.sampling_function = sampling_function
+        self.sampling_parameters = sampling_parameters
+        self.k_sampling_parameters = k_sampling_parameters
+
+    def copy_classifier(self):
+        # Generate synthetic data
+        X_generated = self.sampling_function(
+            *self.sampling_parameters, **self.k_sampling_parameters
+        )
+        df_generated = labeling(X_generated, self.original)
+
+        # Copy the classifier
+        self.copy.fit(df_generated.features, df_generated.labels)
```

Comment:
The idea was precisely not to save the generated data used for the training (for space reasons, because it could be huge), and have a function that would return the same data if the training data was needed for some reason. The get_training_data is using the same seed that was used on creating the copy instance, so it should regenerate the same training data.

Reply:
As discussed last time, not caching it in the object is fine. However, I do think it would make more sense to return the data here since you already have it. That way, if the user wants it, they can save capture it when calling this method rather than having to recompute it. I would expect a user using this tool would already have an idea of whether or not they need to keep the generated data around as a part of their analysis.

Reply:
I've added the boolean parameter `get_training_data` so that the method `copy_classifier` returns the training data when `True`.